libraries/tock-register-interface/README.md

```diff
@@ -190,6 +190,164 @@ register_bitfields! [
 ]
 ```
 
+This generates the modules `Control`, `Status`, and `InterruptFlags`
+    
+```rust
+
+mod Control {
+    pub struct Register;
+    impl ::core::clone::Clone for Register {
+        #[inline]
+        fn clone(&self) -> Register { *self }
+    }
```

Comment:
I think the implementation here is not particularly interesting (in fact, it should always be pretty boring and straightforward for implementations of `Clone`), hence I'd suggest eliding it.